cmd/minishift/cmd/hostfolder/util.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright (C) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hostfolder
+
+import (
+	"fmt"
+	cmdConfig "github.com/minishift/minishift/cmd/minishift/cmd/config"
+	"github.com/minishift/minishift/pkg/minishift/config"
+	"github.com/minishift/minishift/pkg/minishift/hostfolder"
+	"github.com/minishift/minishift/pkg/util"
+	"github.com/minishift/minishift/pkg/util/os/atexit"
+	"github.com/spf13/viper"
+	"regexp"
+	"strings"
+)
+
+const (
+	HostfoldersDefaultPath  = "/mnt/sda1"
```

Comment:
This hard-coded disk path not going to work with hyperkit or xhyve when using raw disk and it changes to `/mnt/vda1`.  As per code, this is only used when a user try to use interactive mode. Can we not add this as part of default string and let user provide the complete path? We can, however, convey some way to use /mnt/sda1 or /mnt/vda1 but don't append ourself.

Reply:
This is existing code. I have not changed that, just moved. That said, this is just the default string displayed. It is used when the user just hits enter. The user can at any stage provide a full path. 

It makes sense to give the user some hint on where he might want to mount to. I think the docs could be extended in this area.

If this particular suggestion causes issues, I suggest creating a follow-up issue. Either we don't display any default at all or do something specific depending on the VM. Right now, I don't see this as a directly related.

Reply:
Originally suggesting was to use `/mnt/data` instead of `/mnt/sda1`, but this was never addressed. Maybe it is now a good time to address this... in a new issue.

Reply:
@gbraad agree. We should have a follow up issue for this.